github3/github.py

```diff
@@ -996,7 +996,8 @@ def pubsubhubbub(self, mode, topic, callback, secret=''):
         :returns: bool
         """
         from re import match
-        m = match('https?://[\w\d\-\.\:]+/\w+/[\w\._-]+/events/\w+', topic)
+        m = match('https?://[\w\d\-\.\:]+/\w+[\w-]+\w/[\w\._-]+/events/\w+',
```

Comment:
It's unlikely that they'll change this. Fixing it later is also always easy and I'd like to not be too relaxed in the data we accept.